static/app/views/nav/primary/components.tsx

```diff
@@ -229,29 +249,135 @@ const baseNavItemStyles = (p: {isMobile: boolean; theme: Theme}) => css`
   `}
 `;
 
-const NavLinkIconContainer = styled('span')`
-  position: relative;
+const ChonkNavLinkIconContainer = chonkStyled('span')`
   display: flex;
   align-items: center;
   justify-content: center;
-  width: 36px;
-  height: 32px;
-  border-radius: ${p => p.theme.borderRadius};
-  overflow: hidden;
+  padding: ${space(1)} ${space(1)};
+  margin-bottom: ${space(0.5)};
+  border-radius: ${p => p.theme.radius.lg};
+`;
+
+const NavLinkIconContainer = withChonk(
```

Comment:
@malwilley I've decided to use the `withChonk` component switch and reimplement the components, as it would have otherwise been quite involved and probably very messy to handle the two differences (our button uses the Button component while the current one seems to use the native element).

With the wihtChonk switch, there is a bit more separation and between the two implementations which should also make cleanup later much easier as we can just delete the non chonk components

Reply:
Definitely agree with this approach!